clique_wp.sh

```diff
@@ -136,7 +127,7 @@ wp post delete 1 --force
 wp comment delete 1 --force
 
 # If they want to disable comments, we'll also disable trackbacks and pings
-if [ "$disableComments" = 'y' ]
+if [[ "${disableComments}" == "y" ]]
```

Comment:
Usually `=` is used to assign variables. If you want to check for equality, use `==`. I'm aware that it still works with `=`, I'm not sure why, though ;)